vllm/model_executor/layers/linear.py

```diff
@@ -1052,16 +1052,30 @@ def weight_loader(self, param: Parameter, loaded_weight: torch.Tensor):
         assert param_data.shape == loaded_weight.shape
         param_data.copy_(loaded_weight)
 
-    def weight_loader_v2(self, param: BasevLLMParameter,
-                         loaded_weight: torch.Tensor):
+    def weight_loader_v2(self,
+                         param: BasevLLMParameter,
+                         loaded_weight: torch.Tensor,
+                         loaded_shard_id: Optional[int] = None):
 
         # Special case for loading scales off disk, which often do not
         # have a shape (such as in the case of AutoFP8).
         if len(loaded_weight.shape) == 0:
             assert loaded_weight.numel() == 1
             loaded_weight = loaded_weight.reshape(1)
 
-        param.load_row_parallel_weight(loaded_weight=loaded_weight)
+        if loaded_shard_id is None:
```

Comment:
I don't quite follow why changing this function is needed. Isn't this function `RowParallelLinear.weight_loader_v2`? So we should always have `loaded_shard_id=None`

Reply:
@ElizaWszola same for this

Reply:
Ok, turns out I don't need it - I must have had a bug in my older code that caused a need for this, but looks like I have fixed it in the meantime